Examples/Scripts/Python/telescope_simulation.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+
+import acts
+import acts.examples
+from acts.examples.simulation import (
+    addParticleGun,
+    EtaConfig,
+    PhiConfig,
+    ParticleConfig,
+    addFatras,
+    addGeant4,
+)
+
+u = acts.UnitConstants
+
+if "__main__" == __name__:
+    detector, trackingGeometry, decorators = acts.examples.TelescopeDetector.create(
+        bounds=[200, 200], positions=[30, 60, 90, 120, 150, 180, 210, 240, 270]
+    )
+
+    field = acts.ConstantBField(acts.Vector3(0, 0, 2 * u.T))
+
+    outputDir = Path.cwd() / "telescope_simulation"
+    if not outputDir.exists():
+        outputDir.mkdir()
+
+    for geant, postfix in [(False, "fatras"), (True, "geant4")]:
+        rnd = acts.examples.RandomNumbers()
```

Comment:
Maybe create the rnd object outside of the for-loop? And initialize with static seed like 42 to have reproducibility (or just make clear we have a static seed here, I'm not sure how default initialization is actually...)?